### ISSUE TITLE
Android SDK : "AppData" not listed

### ISSUE THREAD

**Issue by payne911** [tommyettinger/gdx-liftoff#31]
When trying to find my Android SDK through the File Explorer provided by ``liftoff``, I can't find the ``AppData`` folder. I do see ``Application Data``, but that folder is empty in the visualizer.

My SDK is located in ``C:/Users/payne/AppData/Local/Android/Sdk``, that's slightly problematic...

I can, however, just paste that value in and it gets recognized as valid.

**Comment by tommyettinger** [comment]
AppData is a hidden folder by default on Windows, and apparently VisUI's file picker doesn't show hidden folders. The problem seems to be here in VisUI's source, https://github.com/kotcrab/vis-ui/blob/master/ui/src/main/java/com/kotcrab/vis/ui/widget/file/FileChooser.java#L1754 ; it just refuses to show hidden files. VisUI's file picker just isn't great in general, and it would be nice to have a native one (not Swing!), but I just don't know much about that area. I'll leave this open in case you want to reference it in an issue for VisUI, where it would be more relevant.

**Comment by kotcrab** [comment]
You can set custom file filter.

**Comment by tommyettinger** [comment]
Yeah, a custom file filter would be a workaround for an incomplete file picker implementation, though it would work. I'm inclined to move work toward @raeleus ' new standard-scene2d.ui-based interface, and avoid VisUI hacks that would just need to be redone for a new GUI. Waiting on you, @payne911 :smile: ! I'll probably set a custom file filter for the next release, but since copying in the path works, it isn't urgent. Leaving this open as a reminder.

**Comment by kotcrab** [comment]
I'm not actively developing VisUI anymore, anyone can make a PR that will add "show hidden" config for `DefaultFileFilter`. Anyway, if you want native then [nativefiledialog](https://github.com/mlabbe/nativefiledialog) is probably your best bet, it has LWJGL bindings.
